src/components/Homepage/HomeExp/KivaMultiCategoryGrid.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<div class="md:tw-flex md:tw-items-start">
+		<div class="tw-mr-4">
+			<p class="tw-text-h3 tw-font-medium tw-mb-2 tw-text-secondary">
+				I want to support
+			</p>
+			<loan-category-selector-home-exp
+				v-if="combinedLoanChannelData.length > 1"
+				:loan-channels="combinedLoanChannelData"
+				:selected-channel="selectedChannel.id"
+				@handle-category-click="handleCategoryClick"
+			/>
+		</div>
+		<kiva-classic-loan-carousel
+			:new-home-exp="newHomeExp"
+			:is-visible="showCarousel"
+			:loan-ids="selectedChannelLoanIds"
+			:selected-channel="selectedChannel"
+			:show-view-more-card="showViewMoreCard"
+		/>
+	</div>
+</template>
+
+<script>
+import gql from 'graphql-tag';
+import KivaClassicLoanCarousel from '@/components/LoanCollections/KivaClassicLoanCarousel';
+import LoanCategorySelectorHomeExp from '@/components/LoanCollections/HomeExp/LoanCategorySelectorHomeExp';
+
+export default {
+	name: 'KivaMultiCategoryGrid',
+	components: { LoanCategorySelectorHomeExp, KivaClassicLoanCarousel },
+	props: {
+		/**
+		 * Array of loan channel data in an object
+		 * ex. [{ id: 52, shortName: 'some short name' }]
+		* */
+		contentfulLoanChannels: {
+			type: Array,
+			default: () => [],
+		},
+		/**
+		 * Additional display settings
+		 * Possible Options:
+		 * loanLimit: integer that controls how many loans will be loaded for ALL channels
+		* */
+		loanDisplaySettings: {
+			type: Object,
+			default: () => {}
+		},
+		newHomeExp: {
+			type: Boolean,
+			default: false
+		}
+	},
+	data() {
+		return {
+			loanChannelData: [],
+			selectedChannel: {},
+			showCarousel: false,
+			isUrgencyExpVersionShown: false
+		};
+	},
+	computed: {
+		loanChannelsWithUrgencyExperiment() {
```

Comment:
You can remove this computed property, this experiment won't ever be on this carousel